lib/bug_report_templates/rails_5_master.rb

```diff
@@ -0,0 +1,109 @@
+begin
+  require 'bundler/inline'
+rescue LoadError => e
+  STDERR.puts 'Bundler version 1.10 or later is required. Please update your Bundler'
+  raise e
+end
+
+gemfile(true) do
+  source 'https://rubygems.org'
+
+  gem 'rails', require: false
+  gem 'sqlite3'
+  gem 'activeadmin', git: 'https://github.com/activeadmin/activeadmin', require: false
+  gem 'inherited_resources', require: false
```

Comment:
I'm sorry @senid231, I wasn't clear enough. We do need a specific version for Rails 5 support, but we don't need to point to master. So the `~> 1.7` part is actually needed.